eng/pipelines/templates/scripts/verify-path-length.yml

```diff
@@ -22,16 +25,17 @@ steps:
         print('Analyzing length of paths...')
         for root, dirs, files in os.walk('{0}'.format(source_directory)):
             for file in files:
-                file_path = os.path.join(root, file)
-                if (len(file_path) > longest_file_path_length):
-                    longest_file_path_length = len(file_path)
+                file_path = os.path.relpath(os.path.join(root, file), source_directory)
+                if ((len(file_path) + BasePathLength) > longest_file_path_length):
```

Comment:
I don't think this is going to work you will need to translate between DevOps and pythong. I think you need to change this to `${{ parameters.BasePathLength }}` similar to what you did with source directory.

Reply:
Thanks, Fixed